packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec

```diff
@@ -21,13 +21,14 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}%{?prerelease}.gem
 # start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby >= 1.9
-Requires: %{?scl_prefix_ruby}ruby(rubygems) > 1.3.1
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(faraday) >= 0.14.0
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1.0
```

Comment:
This seems redundant with the next line, was the specfile generated by obal or manually? (i.e. is this a mistake in the gemspec?)

Reply:
it was auto-generated using 'bump_rpm.sh'.  Looks like this is a quirk of the spec file: https://rubygems.org/gems/pulp_2to3_migration_client

Reply:
huh, odd. I tried to find where the gemspec is but seems like it's generated in some automated process as part of the version release. i couldn't find where it got these requirements from.

Reply:
Oh yeah, if the gem has both >= and ~> in the spec, this looks like what you'd get.

Not nice, but also not hurting.